_data/top_nav.yml

```diff
@@ -8,6 +8,12 @@ items:
       -
         label: OpenSearchCon 2023 CFP!
         url: /opensearchcon2023-cfp.html
+  -
+    label: Platform
```

Comment:
Can we move this over to where 'Live Demo!' is on the top-nav, then link to 'Live Demo' underneath? 

Platform
 --> Vector Database
 --> Live Demo
 
thanks!

Reply:
This is resolved.